src/cozy/wrappers/installation-page/installation-page.component.ts

```diff
@@ -101,9 +116,9 @@ export class InstallationPageComponent extends AngularWrapperComponent {
     /* Render */
     /**********/
 
-    protected renderReact() {
+    protected async renderReact() {
         ReactDOM.render(
-            React.createElement(InstallationPage, this.getProps()),
+            React.createElement(InstallationPage, await this.getProps()),

```

Comment:
Interesting. What is the behavior? Does it mount the component and call the constructor at first and then when getProps is finished call willMount then render? 

Or does it mount the component with no props, and then call didUpdate ?

Reply:
This is equivalent to :
```js
const props = await this.getProps()
React.createElement(InstallationPage, props)
```
So your first assertion is the correct one.